velox/parse/DuckLogicalOperator.h

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/external/duckdb/duckdb.hpp"
```

Comment:
Not sure I get it. Are these two files a drop-in copy of DuckDB code? Can't we just include it from our vendored version of their code?

Reply:
The needed plan nodes are defined in duckdb-internal.hpp. I tried including it directly, but it caused conflicts with fmt/format.h. duckdb-internal.hpp includes a copy of that.